arbitrum-docs/stylus/concepts/stylus-gas.md

```diff
@@ -1,20 +1,29 @@
 ---
-title: 'Gas Costs in Stylus'
-sidebar_label: 'Gas Costs in Stylus'
-description: 'A conceptual overview of the pricing models of Stylus, including the relationshup between gas and ink.'
+title: 'Gas and ink (Stylus)'
```

Comment:
intent: drive search traffic about "gas" and "ink" to this doc; drive traffic about "costs" and "pricing" to the reference doc